mmdet3d/models/detectors/imvotenet.py

```diff
@@ -0,0 +1,389 @@
+from torch import nn as nn
+
+from mmdet.models import DETECTORS, build_backbone, build_head, build_neck
+from .base import Base3DDetector
+
+
+@DETECTORS.register_module()
+class ImVoteNet(Base3DDetector):
+    """ImVoteNet model.
+
+    https://arxiv.org/pdf/2001.10692.pdf
+    """
```

Comment:
r"""`ImVoteNet <https://arxiv.org/abs/2001.10692>`_ for 3D detection."""
It seems there exists the same problem in the docstring of VoteNet. Please also modify that accordingly.